plugins/blip/basics/logic/actions.js

```diff
@@ -67,15 +67,21 @@ basicsActions.setSiteChangeEvent = function(sectionName, selectedKey, selectedLa
 
   metricsFunc('Selected ' + selectedLabel);
 
+  var newProfile = _.cloneDeep(this.app.props.patient.profile);
+  if (newProfile.patient.hasOwnProperty('settings')) {
```

Comment:
The `hasOwnProperty` is pretty explicit, but would this situation be a good application for `_.merge` (https://lodash.com/docs/3.10.1#merge)? We might not need the if/else with a `_.merge`